pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -42,31 +41,3 @@ social:
   = view :donor_slider
 
 = view :homepage_video
-
-
-:javascript
-
-  $(document).ready(function() {
-    // For IE8, can't start with opacity in the HTML, so we made things display: none.  Now we show them but
-    // set opacity to 0, ready for code to start showing them by fading them in.
-    $("#petitionexpand").css({display: "block", opacity: 0});
-  });
-
-  $(document).ready(function() {
-    $.ajax({
-      type: "GET",
-      url: '/dashboardapi/user_hero',
-      //data: "id=" + id, // appears as $_GET['id'] @ ur backend side
-      complete: function(data, status) {
-        if (data.responseText == "" || status !== "success") {
-          $("#petitionexpand").animate({opacity:1}, { duration: 2000 });
-          $('#user_hero').css({display: "none"});
-          //$("#petitionaction").animate({opacity:1}, { duration: 2000 });
-        } else {
-          $("#petitionexpand").css({display: "none"});
-          // data is ur summary
-          $('#user_hero').html(data.responseText, status);
-        }
-      }
-    });
-  });
```

Comment:
This ajax call can be removed (and the `#petitionexpand` div initially set to `display: block`) now that users are no longer directed to the homepage when logged-in. `/dashboardapi/user_hero` is implemented with [`head :not_found unless current_user`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/api_controller.rb#L10-L12), so will now always return `:not_found`, so we can skip this ajax step and always start the page with petition block expanded.